test/fixtures/incorrect.js

```diff
@@ -123,17 +131,16 @@ try {
   noop();
 } catch (e) {}
 
-// `no-labels`.
+// `no-labels` and `no-unused-labels`.
```

Comment:
Good point. I used the label in the fixtures so that this particular rule no longer triggers.